guardian/manage-frontend
# Create What Happens Next section and refactor to useAsyncLoader

## What does this change?

This PR adds the 'What Happens Next' section to the Review page of the switch process. It adds a new SVG to represent the new payment information. This PR also uses the new `useAsyncLoader` page to fetch data on every switch page and this page specifically. The review page calls out to the product switch preview endpoint to calculate the amount payable today.



## How to test

Navigate to `/switch/review` and witness the new section. When the page loads it should display the loading spinner with the message 'Loading'.



## How can we measure success?



## Have we considered potential risks?



## Images

![image](https://user-images.githubusercontent.com/114918544/214291787-6956179e-8bec-44be-b87d-97471c915671.png)
![image](https://user-images.githubusercontent.com/114918544/214291900-cda99dfe-f907-4607-aa5b-ff7a92f7459d.png)



## Accessibility
